oeps/best-practices/oep-0051-bp-conventional-commits.rst

```diff
@@ -137,7 +137,11 @@ __ https://open-edx-proposals.readthedocs.io/en/latest/oep-0019-bp-developer-doc
 Footer
 ------
 
-Breaking changes must have a ``BREAKING CHANGE:`` footer in the body.  No other footers are specified.
+Breaking changes must have a ``BREAKING CHANGE:`` footer in the body.
+
+Commits may optionally include the ``the Co-authored-by:`` footer to indicate when the commit is the result of a collaboration.
+
+Private references to content related to the commit may be included using the ``:private-ref:`` footer token.  Private references may only be included in the footer using this token. 
```

Comment:
Earlier it was `private-ref:`, and here it is `:private-ref:`.  I think the first is correct.